lib/phoenix_html/safe.ex

```diff
@@ -70,4 +70,11 @@ end
 
 defimpl Phoenix.HTML.Safe, for: Tuple do
   def to_iodata({:safe, data}), do: data
```

Comment:
I think one reason (maybe the only?) people run into this is that they have the "old" Phoenix book that describes the old way of doing things, but they're generating a new 1.1 app.

Maybe that's going away when the book is updated (if it hasn't already), I dunno.

I feel that this change is small and isolated enough that I think the cost for being possibly-too-helpful is low, but I'm not a maintainer, so it's definitely not my call :)

Let me know where you'd like this to go and I'll make the change. Or feel free to do it yourself outside this PR if you have a specific idea and want to run with it :)